File: Problems/Problem_001/problem_001.cpp

#include<iostream>

using namespace std;

int main()
{
    int n = 1000, sum = 0;
    int i;

    for (i = 1; i < n; i++)
    {
        if(i%3 == 0 || i % 5 == 0)
        {
            sum += i;
        } 
    }

    cout << sum;

    return 0;
}

File: Problems/problem_001.f90
PROGRAM problem_001
    IMPLICIT NONE
    INTEGER :: A = 0, sum = 0 

    Do A = 0, 999
        IF (MOD(A,3) .EQ. 0 .OR. MOD(A,5) .EQ. 0) THEN
            sum = sum + A
        END IF
    END DO
    
    WRITE(*,*) "The sum is: ", sum

END PROGRAM
